package.json

```diff
@@ -1,7 +1,19 @@
 {
```

Comment:
Sorry, doing reviews over the mobile app here. Can you make sure that the indentation is 2 spaces? Seems my suggested change did something to the indentation, but it is hard to verify on mobile. Thanks